meshtastic/tests/test_smoke1.py

```
@@ -1,119 +1,498 @@
+        assert ready_code == 0, ready_output
+        _assert_connected(ready_output)
+        last_output = ready_output
+        if predicate is None or predicate(ready_output):
+            return ready_output
+        remaining = deadline - time.monotonic()
+        if remaining <= 0:
+            break
+        time.sleep(min(INFO_READY_POLL_INTERVAL_SECONDS, remaining))
+    raise AssertionError(
+        f"Device never reached the expected post-mutation state:\n{last_output}"
+    )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)